learning/katas/go/Core Transforms/Combine/CombineFn/pkg/task/task.go

```diff
@@ -47,9 +47,9 @@ func (c *averageFn) MergeAccumulators(accumA, accumB averageAccum) averageAccum
 	}
 }
 
-func (c *averageFn) ExtractOutput(accum averageAccum) float64 {
+func (c *averageFn) ExtractOutput(accum averageAccum) int {
```

Comment:
Actually the ExtractOutput type was correct to use float since average can be non-int.

Reply:
I think you should do this in Go?
```go
return float64(accum.Sum) / float64(accum.Count)
```

Reply:
Henry's right.
`float64(accum.Sum / accum.Count)` will have integer division fraction truncation errors, while `float64(accum.Sum) / float64(accum.Count)` will not.

Go is very specific about requiring explicit conversions between numeric types to avoid this kind of error accidentally.